src/purview.ts

```diff
@@ -1189,57 +914,332 @@ const globalStateTrees: Record<string, StateTree | undefined> = {}
 const globalCSSState: Record<string, CSSState | undefined> = {}
 const DELETE_INTERVAL = 60 * 1000 // 60 seconds
 
-export const reloadOptions = {
-  async saveStateTree(id: string, tree: StateTree): Promise<void> {
-    globalStateTrees[id] = tree
-    if (process.env.NODE_ENV !== "test") {
-      setTimeout(() => this.deleteStateTree(id), DELETE_INTERVAL)
+namespace Purview {
```

Comment:
The following functions/values were moved inside of the `Purview` namespace, mostly corresponding to the previous properties exported on the `default` object:
`createElem`
`handleWebSocket`
`render`
`renderCSS`
`Component`
`scriptPath`
`reloadOptions`

All other functions and values were left outside of the namespace regardless of whether or not they were exported from `purview.ts`. It feels like the `Purview` namespace properly captures the core features of Purview's API and exposes only those elements via the idiomatic `Purview` default import.